### PR TITLE
Added Build Podspec Script

### PR DESCRIPTION
**Pull request by Jake-Prickett**
Added `build_podspec.sh`

### CODE REVIEW

**Comment by Lukasa** [comment]
@swift-nio-bot test this please

File: scripts/build_podspec.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+done
+shift "$((OPTIND-1))"
+
+if [[ $# -eq 0 ]]; then
+  echo "Must provide target version"
+  exit 1
+fi
+
+version=$1
+name="NIOHTTP2"
```
Comment (weissi) [inline scripts/build_podspec.sh]: can we call this `SwiftNIOHTTP2` just to be in line with the other NIO ones.

**Comment by Jake-Prickett** [comment]
@weissi Good catch! Pushed the changes 👍

**Comment by weissi** [comment]
@swift-nio-bot test this please

**Pull request merged**